python/lsst/shapelet/tractor.py

```diff
@@ -42,8 +42,10 @@
 except ImportError:
     import pickle
 
-from .shapeletLib import *
-
+import lsst.pex.exceptions
+from .radialProfile import RadialProfile
+from .multiShapeletBasis import MultiShapeletBasis
+from .shapeletFunction import ShapeletFunction
```

Comment:
Please consider cleaning up two pre-existing issues:
```
try:
    import pickle as pickle
except ImportError:
    import pickle
```
should be simplified to `import pickle` (I am guessing futurize made this bit of silliness).
```
standard_library.install_aliases()
```
needs a `# noqa` to make flake8 happy.

Also as a heads up: this `plotSuite` is almost certainly broken because `y` is not defined when first used. I filed DM-9976 about it.